### PR TITLE
Remove CLI instructions

### PR DESCRIPTION
**Pull request by therealjohn**
CLI support is not working well, and we need to fix the scenario. In the meantime, removing these instructions that get users into a state that doesn't work.

### CODE REVIEW

**Review by eriolchan — APPROVED** [review]

File: msteams-platform/get-started/first-app-blazor.md [hunk 1]
```diff
@@ -41,38 +41,6 @@ Use the Teams Toolkit to create your first project:
    1. Select **Project** > **TeamsFx** > **Configure for SSO...**.
    1. When prompted, sign in to your M365 administrator account.
 
-# [Command line](#tab/cli)
```
Comment (surbhigupta12) [inline msteams-platform/get-started/first-app-blazor.md:44]: Please remove the previous tab

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/first-app-blazor.md](https://github.com/MicrosoftDocs/msteams-docs/blob/932d3388475d7e76e36e84c00c8f53f9fd8e903e/msteams-platform/get-started/first-app-blazor.md) | 86 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/177abd29-51bb-4adc-8b1f-8237eb0db9fa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**